sprint3_API/Advance Sprint3API.postman_collection.json

```diff
@@ -0,0 +1,2374 @@
+{
+	"info": {
+		"_postman_id": "def87d3e-4174-422c-953f-f5bca17f19c0",
+		"name": "Advance Sprint3API",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "33466109"
+	},
+	"item": [
+		{
```

Comment:
Hey Sathiyapriya your task looks good but you are missing schema validation please add this see link  https://www.softwaretestingmaterial.com/json-schema-validation-in-postman/